akka-docs/src/main/paradox/typed/replicated-eventsourcing-db-transport.md

```diff
@@ -183,6 +146,13 @@ To disable this feature you first need to:
 
 The "event publishing" feature publishes each event to the local system event bus as a side effect after it has been written.
 
+## Hot Standby
```

Comment:
This is not relevant for grpc, so I moved it here. For grpc they are always hot standby.